articles/basic-types-and-value-literals.html

```diff
@@ -83,7 +83,7 @@ <h3>Built-in Basic Types in Go</h3>
 
 <p>
 The sizes of <code>int</code> and <code>uint</code> values in memory
-are operation system dependent, either 32 or 64 bits.
+are operating system dependent, either 32 or 64 bits.

```

Comment:
My fault, it should be "are architecture dependent" for preciseness.

Reply:
Maybe, "are operation system and architecture dependent" should be more precise.

Reply:
I have changed this place to the following locally.

```
The sizes of <code>uintptr</code, <code>int</code> and <code>uint</code> values
n memory are implementation-specific.
Generally, The sizes of <code>int</code> and <code>uint</code> values
are 4 on 32-bit architectures, and 8 on 64-bit architectures.
The size of <code>uintptr</code> value must be large enough to
store the uninterpreted bits of any memory address.
```

Reply:
I would change `sizes` to `size`.